Rethinking_2/Chp_07.ipynb

```diff
@@ -0,0 +1,3517 @@
+{
```

Comment:
<p>We can globally set the credible_interval value using <code>az.rcParams['stats.credible_interval'] = 0.89</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Done.